core-bundle/src/Resources/contao/forms/FormFileUpload.php

```diff
@@ -74,6 +74,15 @@ public function __set($strKey, $varValue)
 				}
 				break;
 
+			case 'extensions':
+				if ($varValue)
+				{
+					$strAccept = '.' . implode(',.', StringUtil::trimsplit(',', strtolower($varValue)));
+					$this->arrAttributes['accept'] = $strAccept;
```

Comment:
Since `$strAccept` is not used anywhere else, this should be combined into one line:

```php
$this->arrAttributes['accept'] = '.' . implode(',.', StringUtil::trimsplit(',', strtolower($varValue)));
```

Reply:
You are right. Changed it.